src/components/DeckMechanisms/play.js

```diff
@@ -15,9 +15,15 @@ export const DEFAULT_PLAY_OPTIONS = {
  * @param {Boolean} [options.discard = false] - Whether the play is actually a discard
  * @param {Boolean} [options.free = false] - Whether the play is for free
  * @param {String} [options.mode = 'AUTOMATIC'] - Game mode (MANUAL or AUTOMATIC)
+ * @param {Object} harvestersActions - Reference & Method used to show Harvester’s Dialog
  * @return {Object} Mutated state
  */
-const play = (state, card, options = DEFAULT_PLAY_OPTIONS) => {
+const play = (
+  state,
+  card,
+  options = DEFAULT_PLAY_OPTIONS,
+  harvestersActions
```

Comment:
Maybe we pass it as part of `options`? What do you think?

Reply:
I didn't want to do it this way since it's not really an "option" but I guess we could rename it to pass both in one object